internal/commands/pull/inputs.go

```diff
@@ -94,7 +95,18 @@ func (i *inputs) resolveRemoteApp(ui terminal.UI, clients cli.Clients) (realm.Ap
 	return app, nil
 }
 
-func (i *inputs) resolveClientTemplates(ui terminal.UI, realmClient realm.Client, groupID, appID string) (map[string]*zip.Reader, error) {
+type clientTemplate struct {
+	id     string
+	client *zip.Reader
```

Comment:
[naming nit] not in love with the `client` field of a `clientTemplate` struct... maybe we can call this `pkg` or `zipPkg` or something along those lines?  i think that ultimately reads better above with `namedClient.pkg` (in the for loop)

Reply:
ok so I was thinking the other way, with renaming the field to `pkg` and the struct remaining `clientTemplate` ... I don't want to spend too much effort on this though so feel free to leave as-is, but with a name like `pkg` I think I would at least recommend adding a docstring here to help identify what it's doing

Reply:
oh man.. i really need some more coffee today... not sure why i misinterpreted your initial comment but I reverted the last change and updated the `client`  field name to `zipPkg`